trussworks/next-graphql-fe
# Rename "Case" to "Incident"

## Description

This PR renames all instances of the word "case" - referring to a case of insider threat - to "incident" instead. I don't do anything else in this PR - it's all just a renaming.

**NOTE: This PR depends on the backend changes being merged first: https://github.com/trussworks/django-graphql-be/pull/5 and must be reviewed afterwards.**

## Testing

Everything in this repo should function in exactly the same way as it did before. To test this, we'll want to run all the server and check that all the tests are still working:

1. Run `yarn install` if you might be out of date.
2. Run `yarn dev` to serve the interface.
3. Navigate to the backend repo and start the server there. The e2e tests will try to hit the backend server to test retrieving data.
4. Back in the frontend folder, run `yarn test` to execute the complete suite of tests.
5. If everything looks good, you're done!
